armi/bookkeeping/db/database3.py

```diff
@@ -673,13 +673,23 @@ def loadCS(self):
 
         cs = settings.Settings()
         cs.caseTitle = os.path.splitext(os.path.basename(self.fileName))[0]
-        cs.loadFromString(self.h5db["inputs/settings"][()])
+        try:
+            cs.loadFromString(self.h5db["inputs/settings"][()])
+        except:
```

Comment:
make pylint broad exception exception and explain when it's expected and why it's ok

Reply:
Turns out it should be a KeyError and i was just lazy. Added with comment